File: Shared/Services/UC/UCService.cs
namespace Shared.Services.UC
{
public class UCService : IService
{
    public RadixUCService Radix { get; set; } = new();
    public CurrencyUCService Currency { get; set; } = new();
    public TemperatureUCService Temperature { get; set; } = new();
    public LinkZipUCService LinkZip { get; set; } = new();
    public CommonUCService Common { get; set; } = new();
    public RankUCService Rank { get; set; } = new();
}
}

File: Web/src/services/uc/UCService.ts
import { CommonUCService } from "./CommonUCService";
import { CurrencyUCService } from "./CurrencyUCService";
import { LinkZipUCService } from "./LinkZipUCService";
import { RadixUCService } from "./RadixUCService";
import { RankUCService } from "./RankUCService";
import { TemperatureUCService } from "./TemperatureUCService";

export class UCService implements IService {
  radix = new RadixUCService();
  currency = new CurrencyUCService();
  temperature = new TemperatureUCService();
  linkZip = new LinkZipUCService();
  common = new CommonUCService();
  rank = new RankUCService();
}
